### ISSUE TITLE
Create replacement template for all-auth signup

### ISSUE THREAD

**Issue by nuwang** [galaxyproject/cloudlaunch#102]
If you attempt to signup with social auth, and a user account with that e-mail account already exists, an untemplated page is displayed to the user. This should be replaced with a properly themed page. See: https://github.com/galaxyproject/cloudlaunch/issues/100 for context.

**Comment by thirdknife** [comment]
Will the template replacement apply to the UI project written in angular?

**Comment by nuwang** [comment]
Yes, hitting that django template url should probably just cause a redirect to an appropriate path in the Angular Singular Page application, where the new template will reside. As long as the path doesn't reside in: https://github.com/galaxyproject/cloudlaunch-ui/blob/master/proxy.conf.json, the SPA should get priority in handling it.
